helpers/make_network_dirs_main.py

```diff
@@ -0,0 +1,118 @@
+"""Script that creates all directories"""
+import os
+import sys
+sys.path.append("D:/research-and-development")
```

Comment:
instead of this line (it will be different for all of us), move line 9 above here and use this instead in the append.
`my_wd = os.getcsd()`